test/sentry_test.dart

```diff
@@ -32,8 +31,8 @@ void main() {
       String postUri;
       Map<String, String> headers;
       List<int> body;
-      when(httpMock.post(any, headers: any, body: any))
-          .thenAnswer((Invocation invocation) {
+      httpMock.answerWith((Invocation invocation) {
+        if (invocation.memberName == #close) return null;
```

Comment:
Can we assert after this line that `memberName` is `#post`? Also, let's use `{ }` for the `if` block so that the `return` statement gets its own line.